### PR TITLE
fix: incorrect import path for SSOSilentRequest

### PR DESCRIPTION
**Pull request by Gwash3189**
Hi!

When running the `TypescriptTestApp2.0` sample, `npm start` was failing on an incorrect import path for the `SsoSilentRequest` object. This PR fixes that import path, and gets `npm start` working again. 

### Before

```sh
~/code/scentregroup/microsoft-authentication-library-for-js/samples/msal-browser-samples/TypescriptTestApp2.0 dev
❯ npm run start

> typescript-test-app-2.0@1.0.0 prestart
> npm run clean && npm run build && npm run copy-views


> typescript-test-app-2.0@1.0.0 clean
> npm run clean:build && rimraf dist


> typescript-test-app-2.0@1.0.0 clean:build
> rimraf build-tsc build-babel


> typescript-test-app-2.0@1.0.0 build
> tsc && npm run build:babel && npm run build:webpack && npm run clean:build

src/AuthModule.ts:3:34 - error TS2307: Cannot find module '@azure/msal-browser/dist/src/request/SsoSilentRequest' or its corresponding type declarations.

3 import { SsoSilentRequest } from "@azure/msal-browser/dist/src/request/SsoSilentRequest";
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

### After

```sh
❯ npm run start

> typescript-test-app-2.0@1.0.0 prestart
> npm run clean && npm run build && npm run copy-views


> typescript-test-app-2.0@1.0.0 clean
> npm run clean:build && rimraf dist


> typescript-test-app-2.0@1.0.0 clean:build
> rimraf build-tsc build-babel


> typescript-test-app-2.0@1.0.0 build
> tsc && npm run build:babel && npm run build:webpack && npm run clean:build


> typescript-test-app-2.0@1.0.0 build:babel
> babel build-tsc --out-dir build-babel --source-maps

Successfully compiled 6 files with Babel (314ms).

> typescript-test-app-2.0@1.0.0 build:webpack
> webpack

#... webpack output


Listening on port 30662...
```

### CODE REVIEW

File: samples/msal-browser-samples/TypescriptTestApp2.0/src/AuthModule.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { PublicClientApplication, SilentRequest, AuthenticationResult, Configuration, LogLevel, AccountInfo, InteractionRequiredAuthError, RedirectRequest, PopupRequest, EndSessionRequest } from "@azure/msal-browser";
 import { UIManager } from "./UIManager";
-import { SsoSilentRequest } from "@azure/msal-browser/dist/src/request/SsoSilentRequest";
+import { SsoSilentRequest } from "@azure/msal-browser/dist/request/SsoSilentRequest";
```
Comment (derisen) [inline samples/msal-browser-samples/TypescriptTestApp2.0/src/AuthModule.ts]: wouldn't it be cleaner to import `SsoSilentRequest `along with the rest above (ln 1)?

Comment (jasonnutter) [reply samples/msal-browser-samples/TypescriptTestApp2.0/src/AuthModule.ts]: Agreed, should be imported from the main export on line 1.

Comment (Gwash3189) [reply samples/msal-browser-samples/TypescriptTestApp2.0/src/AuthModule.ts]: @derisen @jasonnutter fair shout! I had blinders on while looking at the file. Addressed in the latest changes.

**Review by jasonnutter — APPROVED** [review]

**Review by derisen — APPROVED** [review]
@Gwash3189 thanks for catching this!

**Pull request merged**